src/app/lib/utilities/usePrevious/index.test.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import usePrevious from '.';
+
+describe('usePrevious', () => {
+  let value;
+  const initialProp = 'first';
+
+  const ComponentWithHook = ({ prop }) => {
+    value = usePrevious(prop);
+    return null;
+  };
+
+  const wrapper = mount(<ComponentWithHook prop={initialProp} />);
+
+  it('should set initial value to null', () => {
+    expect(value).toBe(null);
+    wrapper.setProps({ prop: 'second' });
```

Comment:
Having this statement here means the below test is dependent on this test passing, which I think is incorrect. It should be moved from here and put into the assertion below.
```suggestion
```

Reply:
thanks @rhell